### PR TITLE
Split up the E2EE post into parts

### PR DESCRIPTION
**Pull request by MesterMan03**
- [x] I have tested my changes on my device and it is working.
- [x] I am not using insecure packages.
- [x] I am not adding malicious code that may harm the privacy or security of the reader.

Resolves: #

This PR splits up the "Inside of E2EE" post into parts, effectively turning it into a series. It's also adding new content, which is still in draft stage.

### CODE REVIEW

**Comment by noClaps** [comment]
Breaking change: the package manager for the repo has been changed to Bun. You'll need to merge the changes from main and use Bun from now on.

File: packages/blog/public/files/src/env.d.ts
Comment (noClaps) [inline packages/blog/public/files/src/env.d.ts:1]: Why was this file added?

Comment (MesterMan03) [reply packages/blog/public/files/src/env.d.ts:1]: Weird, I don't remember ever creating the file, it just appeared after I merged main.
I guess I'll remove it.

**Comment by MesterMan03** [comment]
Actually wait, there's a nasty error that I've just found.

**Review by noClaps — APPROVED** [review]
